apps/parse/lib/parse/trip_updates.ex

```diff
@@ -24,6 +24,7 @@ defmodule Parse.TripUpdates do
       trip_id: copy(update.trip.trip_id),
       route_id: copy(update.trip.route_id),
       direction_id: update.trip.direction_id,
+      vehicle_id: vehicle_id(update.vehicle),
```

Comment:
codecov says this line isn't covered

Reply:
Should be better now. Previously this whole file wasn't tested.